app/views/gardens/_plantings.html.haml

```diff
@@ -0,0 +1,3 @@
+- if !plantings.empty?
```

Comment:
Please use `unless plantings.empty?` or `if plantings.present?`. 

`empty?` and `present?` aren't exact inverses of each other, by the way. Please check this [SO question](http://stackoverflow.com/questions/885414/a-concise-explanation-of-nil-v-empty-v-blank-in-ruby-on-rails) if you're not familiar with the details.